spec/views/pages/incompany.html.erb_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe "Render workshops" do
+  let(:workshop1) { create :workshop, title: "Cool Workshop" }
+
+
+  it "shows workshop" do
+
```

Comment:
You're not rendering. You're just testing a model here. So, either move it to `spec/models`  and don't call it 'Render workshops', or create an actual view test (or integration test) and see if you render the right things (that would be a title and a description, from the look of your view)